chihacknight/decarbonize-my-state
# Make State Title and Back Button Sticky

## Overview

Makes the state name and back button on the state details page sticky so that you can easily see what state you are on and go back to the map at any point.

| Desktop | Mobile |
| --- | --- |
| ![Peek 2022-05-03 22-15](https://user-images.githubusercontent.com/3187531/166618845-e4fbd498-6ebc-428e-b6a1-3932ea258feb.gif) | ![Peek 2022-05-03 22-16](https://user-images.githubusercontent.com/3187531/166618847-760a8e28-2aa0-4d9d-a765-17c2aa792cf4.gif) |

Closes #66  

### Demo

See Overview.

### Notes

None.

## Testing Instructions

- Test on desktop confirming you can scroll and see the state title (include with long titles like District of Columbia)
